### PR TITLE
Simplify quickstart by getting rid of cloning manifests repository

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by khanhtc1202** [comment]
Nice
/lgtm

**Comment by khanhtc1202** [comment]
could you update `/docs-dev` and other versions docs (at least v0.21.x) 🙏 If not, when we release the new version docs, this change will be revert since the `/docs` is synced from `/docs-dev`. Basically, we should avoid change on `/docs` directly.

**Comment by nghialv** [comment]
Oops. Let me update them.

File: docs/content/en/docs-v0.18.x/quickstart/_index.md [hunk 1]
```diff
@@ -14,23 +14,16 @@ This guides you to install PipeCD in your kubernetes and deploy a `helloworld` a
...
-```
-
-### 2. Installing control plane
+helm repo add pipecd https://charts.pipecd.dev
 
-``` console
-helm -n pipecd install pipecd ./manifests/pipecd --dependency-update --create-namespace \
-  --values ./quickstart/control-plane-values.yaml
+helm install pipecd pipecd/pipecd -n pipecd --dependency-update --create-namespace \
+  --values https://raw.githubusercontent.com/pipe-cd/manifests/{{< blocks/latest_version >}}/quickstart/control-plane-values.yaml
```
Comment (nghialv) [inline docs/content/en/docs-v0.18.x/quickstart/_index.md:23]: This will show `v0.21.1` but let me fix that problem later.

**Comment by nghialv** [comment]
I have just updated the docs. PTAL.

**Comment by nakabonne** [comment]
Good job
/lgtm

**Comment by khanhtc1202** [comment]
Nice
/approve

**Pull request merged**